site/_posts/2017-09-19-lwimiq-all-hands-busy.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Last Week in ManageIQ: All Hands Busy"
+author: cybette
+date: 2017-09-19
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+Last week, all engineering hands were busy in ManageIQ Engineering All-Hands meeting. A few stray hands still found the opportunity to submit some PRs, but there were no available fingers for typing up an entry for ["Last week in ManageIQ"][lwimiq tag]. So please allow me, [Carol][carol github], to paw in a few words in this edition.
```

Comment:
The `lwimiq tag` link returned a 404

Reply:
it's a relative URL, should be ok in production (it works in my dev env)